channels.yaml

```diff
@@ -550,3 +556,9 @@ releases:
       disable-default-registry-endpoint:
         type: boolean
     featureVersions: *featureVersions-v1
+  - version: v1.27.11+k3s1
```

Comment:
Same comment as for RKE2 versions like putting `1.27.11+k3s1` above 1.28.x definition and changing `v1.28.6+k3s2` to `v1.28.7+k3s1`.